### ISSUE TITLE
limitToFirst and limitToLast are not supported

### ISSUE THREAD

**Issue by chetbox** [soumak77/firebase-mock#28]
I run into an error when testing code that uses `limitToFirst`.

```
TypeError: ref.limitToFirst is not a function
```

My code looks like this:

```javascript
ref.orderByKey()
.startAt('abc')
.limitToFirst(10)
.once('value')
.then( /* ... */ );
```

**Comment by chetbox** [comment]
I can see there is a `limit` function. Is this from a different version of the API to the one I'm using?

**Comment by soumak77** [comment]
@chetbox yeah the library started out as supporting firebase v2 and slowing changes have been made to upgrade it to v3.  Looks like the limit apis need to be updated next.

**Comment by soumak77** [comment]
@chetbox changes are available in v1.0.10

**Comment by chetbox** [comment]
Amazing, thanks!
